gravatar-quickeditor/src/test/java/com/gravatar/quickeditor/ui/alttext/AltTextViewModelTest.kt

```diff
@@ -117,22 +139,23 @@ class AltTextViewModelTest {
 
         viewModel.onEvent(AltTextEvent.AvatarAltTextChange(newAltText))
 
-        viewModel.onEvent(AltTextEvent.AvatarAltTextSaveTapped)
+        advanceUntilIdle()
 
         viewModel.uiState.test {
-            skipItems(2)
+            expectMostRecentItem()
+            viewModel.onEvent(AltTextEvent.AvatarAltTextSaveTapped)
```

Comment:
I think this is a bit more future-proof as we are only testing the states after `AvatarAltTextSaveTapped` event without skipping fixed numbers of states.